data.sql

```diff
@@ -11,3 +11,24 @@ VALUES ('Pikachu', '2021-01-07', 15.04, false, 1);
 
```

Comment:
## 1. Put your project name in the desctiption
![image](https://user-images.githubusercontent.com/93039359/213638169-1395840f-1b8d-4436-a5b6-c78be94b2263.png)

## 2. Put your comment in this section
![image](https://user-images.githubusercontent.com/93039359/213638286-8d694efe-c21d-42b4-aa2f-ff954025ca69.png)

- [ ] Kindly add the information in these sections which are shown in the image above :+1: to make your readme descriptive :blush: